### DISCUSSION TITLE
scoop.sh and ddl

### DISCUSSION THREAD

**Discussion by couleurm** [ScoopInstaller/scoopinstaller.github.io#74] [Show and tell]
hey, I made my own website to browse scoop packages, you're free to take any of the features your website doesnt have 👍

http://couleurm.github.io/ddl

https://github.com/couleurm/ddl

**Comment by rashil2000** [comment]
Interesting idea! 

I gave it a try - searched for hyperfine, and clicked on the search result - it opened a page saying `Redirecting to <url>...` but then nothing happened.

**Comment by couleurm** [comment]
yeah i'm not sure how to ensure the download has started so i just wait 1000ms, it might not be enough for small connections

if yall got better js code for that, feel free to PR

here's how it's suppose to behave:

https://github.com/ScoopInstaller/scoopinstaller.github.io/assets/82747632/596798e5-35be-4e12-9a56-abd812dc4dd3
